neurom/apps/tests/test_morph_stats.py

```diff
@@ -27,7 +27,7 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import os
-from nose import tools as nt
+from nose.tools import assert_equal, ok_, assert_true, eq_, assert_almost_equal, assert_dict_equal, assert_raises, assert_greater_equal
```

Comment:
may as well take this opportunity to use either `assert_equal` or `eq_`, instead of a mishmash